api/client/service/opts.go

```diff
@@ -199,14 +198,13 @@ func (m *MountOpt) Set(value string) error {
 			mount.Type = swarm.MountType(strings.ToLower(value))
 		case "source", "name", "src":
 			mount.Source = value
-		case "target", "dst", "dest", "destination", "path":
```

Comment:
`destination` is something people use for `target` in other APIs.

People think of things in different ways and this helps them do that.